### PR TITLE
features: divide api -> api, apiDI modules

### PR DESCRIPTION
**Pull request by PanVova**

### CODE REVIEW

File: apiDI/src/main/java/com/example/apidi/base/BaseApiModule.kt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.panhuk.api.di.base
+package com.example.apidi.base
```
Comment (NickHuk) [inline apiDI/src/main/java/com/example/apidi/base/BaseApiModule.kt]: `com.example` WTF? Please name package `com.panhuk...`

File: repositoryDI/build.gradle [hunk 2]
```diff
@@ -49,5 +49,5 @@ dependencies {
   implementation project(':repositoryImpl')
   implementation project(':dataSource')
   implementation project(':dataSourceDI')
-  implementation project(':api')
```
Comment (NickHuk) [inline repositoryDI/build.gradle:44]: removed `:api` dependencies - well done !

**Review by NickHuk — CHANGES_REQUESTED** [review]

File: apiDI/src/main/java/com/example/apidi/base/BaseApiModule.kt (hunk 1 above)
Comment (PanVova) [reply apiDI/src/main/java/com/example/apidi/base/BaseApiModule.kt]: renamed

**Review by NickHuk — APPROVED** [review]
GJ!

**Pull request merged**